tests/Integration/TestCase.php

```diff
@@ -51,6 +56,26 @@ public function setUp($defaultSerializer = '')
                 'publisher' => 'Bloody Fantasy inc.',
             ],
         ];
+
+	    $this->testPosts = [
```

Comment:
Changed the tests to use the books array. But added a new transformer for the test itself. The new transformer has the includes as default so that we can test the removal of them